nx-X11/programs/Xserver/dix/gc.c

```diff
@@ -1209,6 +1209,7 @@ SetClipRects(GCPtr pGC, int xOrigin, int yOrigin, int nrects,
     (*pGC->funcs->ChangeClip)(pGC, newct, (void *)prectsNew, nrects);
     if (pGC->funcs->ChangeGC)
 	(*pGC->funcs->ChangeGC) (pGC, GCClipXOrigin|GCClipYOrigin|GCClipMask);
+    free(prectsNew);
```

Comment:
IIRC it must be checked what (*pGC->funcs->ChangeClip) is expected to do with prectsNew. It could be that it stores the pointer somewhere.

Reply:
removed